modules/face/include/opencv2/face/face_alignment.hpp

```diff
@@ -0,0 +1,145 @@
+Copyright (C) 2009-2011, Willow Garage Inc., all rights reserved.
+Copyright (C) 2009-2016, NVIDIA Corporation, all rights reserved.
+Copyright (C) 2010-2013, Advanced Micro Devices, Inc., all rights reserved.
+Copyright (C) 2015-2016, OpenCV Foundation, all rights reserved.
+Copyright (C) 2015-2016, Itseez Inc., all rights reserved.
+Third party copyrights are property of their respective owners.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+  * Redistributions of source code must retain the above copyright notice,
+    this list of conditions and the following disclaimer.
+
+  * Redistributions in binary form must reproduce the above copyright notice,
+    this list of conditions and the following disclaimer in the documentation
+    and/or other materials provided with the distribution.
+
+  * Neither the names of the copyright holders nor the names of the contributors
+    may be used to endorse or promote products derived from this software
+    without specific prior written permission.
+
+This software is provided by the copyright holders and contributors "as is" and
+any express or implied warranties, including, but not limited to, the implied
+warranties of merchantability and fitness for a particular purpose are disclaimed.
+In no event shall copyright holders or contributors be liable for any direct,
+indirect, incidental, special, exemplary, or consequential damages
+(including, but not limited to, procurement of substitute goods or services;
+loss of use, data, or profits; or business interruption) however caused
+and on any theory of liability, whether in contract, strict liability,
+or tort (including negligence or otherwise) arising in any way out of
+the use of this software, even if advised of the possibility of such damage.*/
+#ifndef __OPENCV_FACE_ALIGNMENT_HPP__
+#define __OPENCV_FACE_ALIGNMENT_HPP__
+#include "opencv2/core.hpp"
+#include "opencv2/objdetect.hpp"
+#include "opencv2/imgproc.hpp"
+#include <iostream>
+#include <string>
+#include <sstream>
+#include <vector>
+#include <fstream>
+#include <queue>
+#include <algorithm>
+#include <fstream>
+
+namespace cv{
+namespace face{
+class FacemarkKazemi
+{
+public:
+        /** @returns The number of landmarks detected in an image */
+        virtual int getNumLandmarks() const = 0;
+        /** @returns The number of faces detected in an image */
+        virtual int getNumFaces() const = 0;
+        /** @brief This function extract the data for training from .txt files which contain the corresponding image name and landmarks.
+        *Each file's first line should give the path of the image whose
+        *landmarks are being described in the file.Then in the subsequent
+        *lines there should be coordinates of the landmarks in the image
+        *i.e each line should be of the form "x,y"(ignoring the double quotes)
```

Comment:
lets simply remove the quotes instead of actually telling people to ignore them.